src/main/java/th/ac/kmitl/science/comsci/example/nameprinters/NamePrinterBaseExampleNamePrinter.java

```diff
@@ -0,0 +1,8 @@
+package th.ac.kmitl.science.comsci.example.nameprinters;
+
+public class NamePrinterBaseExampleNamePrinter extends NamePrinterBase {
```

Comment:
Why the name is so long?
What does this class stand for?
How can we use it, what the responsibility of this class?

Reply:
It's an example for extending `NamePrinterBase`. It can be safely removed.

Reply:
If it is and example Why should it name "ExampleNamePrinter"

Reply:
@Nagato-Yuki BTW, why not refactor and move your name to new implementation ?

Reply:
OK I'll go with @mahasak's suggested approach. This will be more descriptive.